### ISSUE TITLE
Export TS Types and Interfaces

### ISSUE THREAD

**Issue by swreeser** [mohsinulhaq/react-popper-tooltip#90]
I see the changes I need are merged into master #84, thanks @dimorphic 😻

I propose releasing the latest changes.
https://github.com/mohsinulhaq/react-popper-tooltip/compare/v2.10.1...master
```
v2.10.2 🏆
What Changed?
- update lint preference and dependencies. 🧹
- export TS types & interfaces (#84) 📦
```

**Comment by swreeser** [comment]
Whoops, i tried the Related Issues search and didn't realize it had cleared my message out.

**Comment by mohsinulhaq** [comment]
I am making a new minor feature release soon. This will go with that.

**Comment by mohsinulhaq** [comment]
Published [v2.11.0](https://github.com/mohsinulhaq/react-popper-tooltip/releases/tag/v2.11.0) with the changes you require. Please try it out.
